official/resnet/imagenet_main.py

```diff
@@ -311,20 +312,26 @@ def imagenet_model_fn(features, labels, mode, params):
       num_images=_NUM_IMAGES['train'], boundary_epochs=[30, 60, 80, 90],
       decay_rates=[1, 0.1, 0.01, 0.001, 1e-4], warmup=warmup, base_lr=base_lr)
 
+  dtype = params['dtype']
+  if params['use_keras_model']:
+    print('use keras model')
```

Comment:
We can remove these print statements I think.

Reply:
Done.